docs/spire-helm.md

```diff
@@ -62,6 +63,46 @@ export SPIRESERVER_NS=tornjak
 kubectl create ns $SPIRESERVER_NS
 ```
 
+### Multi-cluster Support
+If you want to setup a SPIRE Server
+that supports multi-cluster deployment
+where SPIRE agents are distributed in several, remote clusters,
+all reporting to a single SPIRE Server,
+please follow the additional steps outlined here,
+otherwise skip to [Helm Deployment](#helm-deployment)
+
+### Capture the remote KUBCONFIG configurations
+SPIRE server attests the remote agents by verifying the KUBECONFIG configuration.
+Therefore, for every remote cluster, other then the cluster hosting the Tornjak,
+we need KUBECONFIG for this cluster.
+
+Gather the KUBECONFIG for every remote cluster
+and output them into individual files
+using  `kubectl config view --flatten`:
```

Comment:
maybe add a warning here that it is important that `--flatten` be done so that it can provide a portable `KUBECONFIG`, just in case people think its just a regular KUBECONFIG file that they have. Maybe you can call them portable `KUBECONFIG` files.